package bankSystem;

public class TestError {
	public static void main(String[] args) {
		int number1 = 0;
		String number2;
		try {
			number2 = number1;
		} catch (ArithmeticException exception) {
			System.out.println("Erro: " + exception);
		}
		System.out.println(result);
	}
}